.env

```diff
@@ -0,0 +1,9 @@
+#ROLLBAR_POST_TOKEN=
+
+API_URL=http://localhost:3000
+UPLOAD_URL=http://localhost:3000
+DATA_URL=http://localhost:3000
+BLIP_URL=http://localhost:3000
+DEBUG_ERROR=false
```

Comment:
For developers, it may be useful to have additional logging and debug options enabled by default?

Reply:
Agreed.

Reply:
Done.